tests/db_functions/tests.py

```diff
@@ -25,6 +25,83 @@ def truncate_microseconds(value):
 
 class FunctionTests(TestCase):
 
+    def test_strindex(self):
```

Comment:
I'd put these in a separate file, e.g. `test_strindex.py`. Use `setUpTestData` if possible.